languages/common-lisp/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,92 @@
+## Comments
+
+As a quick side-note before jumping into things, Common Lisp, like most
+programming languages, allows the programmer to write "comments" that are
+ignored when your program is being run. In Common Lisp, single-line comments
+begin with one or more semi-colons (`;`) and, occasionally, you may see the
+following:
+
+```lisp
+(code...) ; => value
+```
+
+Where the comment is being used to indicate what value is returned by Common
+Lisp after running the code on that line.
+
+## S-Expressions
+
+All Common Lisp code is made from S-Expressions (or sexpr for short). An
+S-Expression is either an "atom" (a single, indivisible value) or a list (also
+termed a "cons"). A cons is made up of two parts: the first element and the rest
+of the elements. For historical reasons these two parts are called the `car` and
+the `cdr`. When S-Expressions are evaluated as code, the first element (`car`)
+represents the function being called while the rest of the elements (`cdr`)
+represent the arguments to that function. In other words:
+
+```lisp
+(<function> <arg1> <arg2> ... <argN>)
+; ^ car ^  |        ^ cdr ^
+```
+
+When S-Expressions like this are evaluated, they automatically return some value
+which takes the place of this expression. When writing your own functions (using
+`defun`), the last value within the body of the `defun` is automatically
+returned. For example, if you define the following function:
+
+```lisp
+(defun gimme-foo () 'foo)
```

Comment:
I'm wondering if perhaps the three code blocks could be combined into one block:

```suggestion
(defun gimme-foo () 'foo)

(gimme-foo) ; => FOO
```

Note that you could consider adding comments to the code samples to, to replace some of the text outside the code blocks.